css-uc/index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en-us">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Exercise 1</title>
+  <link rel="stylesheet" href="styles.css">
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+
+<body>
+  <header>
+
+    <nav class="flex justify-between mt-7 items-center">
+
+      <img src="./assets/images/PlanIT_Logo.jpeg" alt="planIT_Logo" class="img-size-s">
+
+      <ul class="list-none flex p-0 gap-4">
+        <li><a class="text-black no-underline font-light text-b" href="#Caracteristicas">Caracteristicas</a></li>
+        <li><a class="text-black no-underline font-light text-b" href="#Precio">Precio</a></li>
+        <li><a class="text-black no-underline font-light text-b" href="#Contacto">Contacto</a></li>
+      </ul>
```

Comment:
It would be a good idea in this case, and in some other places to make use of css inheritance to avoid repetition of properties and that way make the markup cleaner. Check [this](https://stackoverflow.com/questions/5612302/which-css-properties-are-inherited) link as a reference about which css properties are inherited